HorticraftPricer.ahk

```diff
@@ -464,13 +466,31 @@ valuedictionary[RAW_Jewel_Implicit_Cluster] := Jewel_Implicit_Cluster
             IfInString, itemLine, %rawVal%
             {
                 outPrice := price
+				nrOfMatchingLines+=1
                 Break
             }
         }
 
         ; Append our found price, which will be "-" if no match was found
-        outputText := outputText "/" outPrice
+        outputText := outputText outPrice "/"
+
     }
+	if (nrOfMatchingLines > 0)
```

Comment:
Can just simplify this a bit and re-use the `nrOfMatchingLines` var by doing a simple `while (nrOfMatchingLines <3)` and then `nrOfMatchingLines+=1`, to reduce nesting and trim it up a bit